core/src/main/java/com/ritense/valtimo/camunda/processaudit/ProcessStartedEvent.java

```diff
@@ -30,8 +36,16 @@ public ProcessStartedEvent(
         String user,
         String processDefinitionId,
         String processInstanceId,
-        String businessKey
+        String businessKey,
+        String processDefinitionKey
     ) {
         super(id, origin, occurredOn, user, processDefinitionId, processInstanceId, businessKey);
+        this.processDefinitionKey = processDefinitionKey;
+    }
+
+    @JsonIgnore(false)
```

Comment:
We usually use: @JsonProperty here

I'm not sure if @JsonIgnore(false) also works